deep_qa/data/tokenizers/character_tokenizer.py

```diff
@@ -29,8 +29,8 @@ def index_text(self,
 
     @overrides
     def embed_input(self,
-                    input_layer: 'keras.layers.Layer',
```

Comment:
If we could change a setting in sphinx to make it ok with this kind of annotation, I'd prefer that.  These annotations should work: https://www.python.org/dev/peps/pep-0484/#forward-references.  Or is it that we need a module import statement somewhere?